laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -674,9 +692,9 @@ void LaserScanMatcher::getPrediction(double& pr_ch_x, double& pr_ch_y,
   // **** use velocity (for example from ab-filter)
   if (use_vel_)
   {
-    pr_ch_x = dt * latest_vel_msg_.twist.linear.x;
```

Comment:
This looks correct; `latest_vel_msg_` IS a `Twist` so no need for `twist` member.